api.rst

```diff
@@ -1509,3 +1507,5 @@ PostgREST translates `PostgreSQL error codes <https://www.postgresql.org/docs/cu
 +--------------------------+-------------------------+---------------------------------+
 | other                    | 400                     |                                 |
 +--------------------------+-------------------------+---------------------------------+
+
+.. [#] Since, to stop a function from being universally executable we revoke its `EXECUTE` privilege from the `PUBLIC` virtual role, it would appear that functions typically start off with the `EXECUTE` privilege granted to `PUBLIC`, but that's the apparent truth and not the literal truth. In reality a function will start off with no privileges given to any role at all, and in that case the system behaves `as though` `PUBLIC` were given the function's `EXECUTE` privilege. Therefore if you see a blank “Access privileges” field in the output of `\df+`, `PUBLIC` effectively has the function's `EXECUTE` privilege (even though it was never actually granted to `PUBLIC`). Thereafter, subsequent to any successful `GRANT` or `REVOKE` operation performed on the function, Postgresql will employ its conventional privilege system. Notably the presence of `=X/grantor` indicates that `PUBLIC` has the `EXECUTE` privilege and, conversely, the absence of `=X/grantor` will imply that `PUBLIC` does not have the `EXECUTE` privilege.
```

Comment:
The blank access privileges is so weird. And actually it doesn't necessarily mean PUBLIC. It can also mean only OWNER. For example I can do this:
```sql
create function f() returns text as $$ select 'f' $$ language sql;
\df+ f 
-- Access privileges  blank             
revoke EXECUTE on function f() from PUBLIC;
\df+ f 
-- Access privileges   │ "steve-chavez"=X/"steve-chavez"
revoke EXECUTE on function f() from "steve-chavez" ;
-- Access privileges   blank again

-- the owner("steve-chavez") still has the privilege to execute it
-- but not PUBLIC
begin;
set local role other;
select f();
-- ERROR:  42501: permission denied for function f
rollback;
```
Maybe some mention of blank access privileges being inconsistent can be included.

I think the footnote is valuable, the  `=X/grantor` notation for PUBLIC is really worth mentioning.